subsys/bluetooth/common/Kconfig

```diff
@@ -4,6 +4,190 @@
 # Copyright (c) 2016 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+config BT_BUF_ACL_TX_SIZE
+	int "Maximum supported ACL size for outgoing data"
+	range 27 251
+	default 27
+	help
+	  Maximum supported ACL size of data packets sent from the Host to the
+	  controller. This value does not include the HCI ACL header.
+	  The Host will segment the data transmitted to the controller so that
+	  packets sent to the controller will contain data up to this size.
+	  In a combined build this value will be set in both the Host and the
+	  Controller.
+	  In a Host-only build the Host will read the maximum ACL size supported
+	  by the controller and use the smallest value supported by both the
+	  host and the controller.
+	  The Host supports sending of larger L2CAP PDUs than the ACL size and
+	  will fragment L2CAP PDUs into ACL data packets.
+	  The controller will return this value in the HCI LE Read Buffer
+	  Size command response. If this size if greater than effective Link
```

Comment:
Should we reference V2 here already, or update this text in the V2 PR (https://github.com/zephyrproject-rtos/zephyr/pull/34806)?